packages/beta/src/__tests__/api/monitoringTasksApi.int.spec.ts

```diff
@@ -93,6 +94,32 @@ describe('monitoring tasks api', () => {
     done();
   }, 10000);
 
+  test('upsert monitoring task', async () => {
+    const sessionsRes = await client.post<SessionsResponse>(sessionsApi, {
+      data: {
+        items: [
+          {
+            clientId: process.env.COGNITE_CLIENT_ID,
+            clientSecret: process.env.COGNITE_CLIENT_SECRET,
+          },
+        ],
+      },
+    });
+
+    const response = await client.monitoringTasks.upsert([
+      {
+        externalId: monitoringTaskExternalId,
+        name: monitoringTaskNameUpdated,
+        channelId: channel.id,
+        model: testMtModel,
+        nonce: sessionsRes?.data?.items[0]?.nonce,
+      },
+    ]);
+
+    expect(response.length).toBe(1);
+    expect(response[name]).toBe(monitoringTaskNameUpdated);
+  });
+
```

Comment:
Didn't dive in to this, just for my own curiosity, how are these tests run, are they run against an actual backend or are the responses mocked?

Reply:
They do yea